dashboard/app/mailers/parent_mailer.rb

```diff
@@ -1,4 +1,5 @@
-class ParentMailer < ActionMailer::Base
+class ParentMailer < ApplicationMailer
+  include ActionMailerMetrics
```

Comment:
You don't need this since it's already included in the parent class `ApplicationMailer`